src/sentry/templates/sentry/emails/digests/subject.txt

```diff
@@ -0,0 +1 @@
+Notifications for {{ project.name }}
```

Comment:
I don't have a strong opinion about this (I doubt anybody is actually going to override these templates in practice) but it seemed a little more consistent of an approach and I don't think the expense is really all that significant in the scheme of things.